src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,51 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+};
+
+export const MovieCard: React.FC<Props> = ({
+  title,
+  description,
+  imgUrl,
+  imdbUrl,
+}) => (
```

Comment:
```suggestion
export const MovieCard: React.FC<Props> = (props) => {
const { movie } = props;
```
Let's stick to this syntax for props usage